tests/kafkatest/services/security/security_config.py

```diff
@@ -172,6 +173,52 @@ def kafka_opts(self):
         else:
             return ""
 
+    def set_acls(self, protocol, kafka, zk, topic, group):
```

Comment:
@granders those ACL calls are stateless, you're right, so they can be pretty much anywhere. The main reason to have it there was to concentrate the security stuff in one place, but you're right it might be better to put some more structure into it. I'm fine either way, what do you think @benstopford?